standard/types.md

```diff
@@ -6,10 +6,11 @@ The types of the C# language are divided into two main categories: ***reference
 
 ```ANTLR
 type
-    : reference_type
-    | value_type
+    : value_type
+    | reference_type
+    | nullable_type_parameter
     | type_parameter
-    | pointer_type     // unsafe code support
+    | type_unsafe
```

Comment:
Was this change from `pointer_type` to `type_unsafe` deliberate? Is there another change coming to clause 23?

Reply:
I think that's a merge mistake. Reverted.